test/extensions/filters/http/jwt_authn/authenticator_test.cc

```diff
@@ -121,83 +121,88 @@ TEST_F(AuthenticatorTest, TestJwtWithNonExistKid) {
   auto headers =
       Http::TestHeaderMapImpl{{"Authorization", "Bearer " + std::string(NonExistKidToken)}};
 
-  MockAuthenticatorCallbacks mock_cb;
-  EXPECT_CALL(mock_cb, onComplete(_)).WillOnce(Invoke([](const Status& status) {
+  std::function<void(const Status&)> mock_cb = [](const Status& status) {
     ASSERT_EQ(status, Status::JwtVerificationFail);
-  }));
-
-  auth_->verify(headers, &mock_cb);
+  };
+  auto tokens = filter_config_->getExtractor().extract(headers);
+  auth_->verify(headers, std::move(tokens), std::move(mock_cb));
 }
 
 // This test verifies if Jwt is missing, proper status is called.
 TEST_F(AuthenticatorTest, TestMissedJWT) {
   EXPECT_CALL(mock_factory_ctx_.cluster_manager_, httpAsyncClientForCluster(_)).Times(0);
-  EXPECT_CALL(mock_cb_, onComplete(_)).WillOnce(Invoke([](const Status& status) {
-    ASSERT_EQ(status, Status::JwtMissed);
-  }));
 
   // Empty headers.
   auto headers = Http::TestHeaderMapImpl{};
-  auth_->verify(headers, &mock_cb_);
+  std::function<void(const Status&)> mock_cb = [](const Status& status) {
```

Comment:
Acutally, this is NOT mock.   it should not be called mock_cb.   call it "on_complete_cb"

Reply:
fixed.